pkg/rancher/migrations.go

```diff
@@ -645,3 +651,50 @@ func insertOrUpdateCondition(d data.Object, desiredCondition summary.Condition)
 
 	return true, nil
 }
+
+func rkeResourcesCleanup(w *wrangler.Context) error {
+	cm, err := getConfigMap(w.Core.ConfigMap(), rkeCleanupMigration)
+	if err != nil {
```

Comment:
could we update to `if err != nil || cm == nil {return err}` for code consistency in this file

Reply:
Also remove `rkeClustersAnnotatedForMigrationKey`